ddtrace/contrib/flask/patch.py

```diff
@@ -262,6 +262,8 @@ def unpatch():
 def _wrap_start_response(func, span, request):
     def traced_start_response(status_code, headers):
         code, _, _ = status_code.partition(" ")
+        # If values are accessible, set the resource as `<method> <path>` and add other request tags
+        _set_request_tags(span)
```

Comment:
If we do it here do we actually need to call it again later in `traced_wsgi_app`? Could we be smarter about which tags we set at which time?

Reply:
when we discussed it the idea was: this might be the only place we need to call it, but to be safe we didn't want to remove the setting of tags in the existing place in case there is an edge case we are unaware of.

The way the tags are set will only get updated if they are not already set, so we hit some loop/checking cost, but should be safe to call multiple times per-request.

What do you think?

Reply:
Got it, makes sense for now 👍 

I think the action item is to add a comment for the second call just so that we don't lose this context in the future.